clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/LoadBalancersSpec.java

```diff
@@ -0,0 +1,254 @@
+import com.netflix.spinnaker.clouddriver.ecs.EcsSpec;
+import com.netflix.spinnaker.clouddriver.ecs.provider.EcsProvider;
+import com.netflix.spinnaker.clouddriver.ecs.provider.view.EcsAccountMapper;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import java.util.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(
+    classes = {Main.class},
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource(properties = {"spring.config.location = classpath:clouddriver.yml"})
+public class LoadBalancersSpec extends EcsSpec {
+
+  @Autowired private ProviderRegistry providerRegistry;
+
+  @MockBean EcsAccountMapper mockEcsAccountMapper;
+
+  private final Logger log = LoggerFactory.getLogger(getClass());
+  private final String ECS_ACCOUNT_NAME = "ecs-account";
+  private final String TEST_REGION = "us-west-2";
+
+  @Test
+  public void getLoadBalancersTest() {
+    // given
+    ProviderCache ecsCache = providerRegistry.getProviderCache(EcsProvider.NAME);
+    String testNamespace = Namespace.LOAD_BALANCERS.ns;
+    String loadBalancerKey =
+        com.netflix.spinnaker.clouddriver.aws.data.Keys.getLoadBalancerKey(
+            testNamespace, "*", TEST_REGION, "*", "*");
+
+    Set<String> keys = new HashSet<>();
+    keys.add(
+        "aws:targetGroups:my-aws-devel-acct:us-west-2:spinnaker-ecs-demo-artifacts-tg:ip:vpc-07daae48bf98a8fd8");
+
+    List<String> securityGroups = new ArrayList<>();
+    securityGroups.add("test-security");
+
+    Map<String, Collection<String>> relationships = new HashMap<>();
+    relationships.put("targetGroups", keys);
+
+    Map<String, Object> attributes = new HashMap<>();
+    attributes.put("account", ECS_ACCOUNT_NAME);
+    attributes.put("region", TEST_REGION);
+    attributes.put("name", ECS_ACCOUNT_NAME);
+    attributes.put("vpcId", "vpc-123");
+    attributes.put("loadBalancerType", "test-type");
+    attributes.put("securityGroups", securityGroups);
+    attributes.put("targetGroups", "test-target");
+    attributes.put("loadBalancerName", "testLB");
+
+    DefaultCacheResult testResult =
+        buildCacheResultForLB(attributes, testNamespace, loadBalancerKey, relationships);
+    ecsCache.addCacheResult("TestAgent", Collections.singletonList(testNamespace), testResult);
+
+    String url = getTestUrl("/ecs/loadBalancers");
+
+    when(mockEcsAccountMapper.fromAwsAccountNameToEcsAccountName("*")).thenReturn(ECS_ACCOUNT_NAME);
+
+    // when
+    Response response =
```

Comment:
Is it possible to deserialize this into the actual return object instead as a string?